bmarlow/Week 3/mailroom.py

```diff
@@ -0,0 +1,131 @@
+import os
+
+#build our initial dictionary
+dictionary = {
+	'Ricky': '1000',
+	'Bobby': '1100,2200',
+	'Hansel': '5000,6000,7000',
+	'Mr. Burgandy': '1,2,3,4',
+	'Austin': '123,4567,78,99,100',
+	'Dale': '14,234,878,345,345,234'
+}
```

Comment:
While this works as you have coded it.  it begs for a List of donation values rather than a single string with values separated by a comma. `{ 'Ricky': [1000], 'Bobby': [1100, 2200], ` etc...